# Introdução


Anime é um estilo de animação originário do Japão que abrange uma ampla variedade de gêneros e estilos. A palavra "anime" é uma abreviação da palavra inglesa "animation" e, no Japão, refere-se a qualquer tipo de animação, seja ela japonesa ou não. No entanto, fora do Japão, o termo é comumente usado para se referir especificamente às animações produzidas no Japão ou influenciadas pelo estilo japonês.

O anime abrange uma gama diversificada de temas, desde aventuras épicas e ficção científica até romances, comédias e histórias cotidianas. Os personagens frequentemente apresentam características distintas, como olhos grandes e expressivos, estilos de cabelo únicos e traços faciais distintos. As narrativas podem ser complexas e envolventes, explorando temas profundos e questões filosóficas, ou podem ser leves e destinadas ao entretenimento puro.

Uma característica marcante do anime é a capacidade de atrair públicos de todas as idades. Existem animes voltados para crianças, adolescentes e adultos, e muitos deles conquistam fãs ao redor do mundo. Alguns animes também são adaptados de mangás, que são histórias em quadrinhos japonesas.

Ao longo dos anos, o anime tornou-se uma parte significativa da cultura pop global, influenciando a indústria do entretenimento em diversos países. A comunidade de fãs de anime é vasta e apaixonada, participando de convenções, eventos e compartilhando sua paixão online.


![image.png](https://img.freepik.com/vetores-gratis/ilustracao-de-kitsune-desenhada-a-mao_23-2149376317.jpg?w=740&t=st=1702516178~exp=1702516778~hmac=71a8710d837b9da71ae6b47e3b7899f941f834963059dffa6c139d444c11f3ae)

Descrição das colunas:

* Rank : Posição
* Name: Nome
*Japanese_name: Nome em japonês
*Type: Tipo
*Episodes: Episódios
*Studio: Estudio
*Realese_season: Temporada
*Tags: Etiquetas
*Rating: Avaliação
*Release_year: Ano de lançamento
*End_year: Ano de encerramento
*Description: Descrição
*Content_warning: Aviso de conteúdo sensível
*Related_mange: Manga relacionado(Erro de tipografia)
*Related_anime: Anime relacionado
*Voice_actors: Dubladores
*Staff: Equipe de produção

[Fonte](https://www.kaggle.com/datasets/vishalmane10/anime-dataset-2022)


In [ ]:
import pandas as pd

In [ ]:
df_anime = pd.read_csv ('/content/Anime.csv')

In [ ]:
df_anime.sort_values(by=['Name']).reset_index()

index   Rank                                               Name  \
0       6654   6655  "Anata wo Hitokoto de Arawashite Kudasai" no S...   
1       6838   6839          "Bungaku Shoujo" Kyou no Oyatsu: Hatsukoi   
2       2833   2834                           "Bungaku Shoujo" Memoire   
3       2352   2353                             "Bungaku Shoujo" Movie   
4      13115  13116                                             "Eiji"   
...      ...    ...                                                ...   
18490  13583  13584                                            ēlDLIVE   
18491   5886   5887                              ŹOOĻ: Bang!Bang!Bang!   
18492  18016  18017                                      ​Fuyu no Niji   
18493  18471  18472                                      ​Semi Shigure   
18494  12454  12455                                                  ∞   

                                           Japanese_name   Type  Episodes  \
0       I'm Bad at the Question "Please Describe Your...  Other       NaN   
1                                                    NaN  OVA         1.0   
2                                                    NaN  OVA         3.0   
3                                                    NaN  Movie       NaN   
4                                                    NaN  Movie       NaN   
...                                                  ...    ...       ...   
18490                                                NaN  TV         12.0   
18491                                                NaN  Music       NaN   
18492                                                NaN  Movie       NaN   
18493                                                NaN  Movie       NaN   
18494                                                NaN  Web         1.0   

                        Studio Release_season  \
0                      Sunrise            NaN   
1               Production I.G            NaN   
2               Production I.G            NaN   
3               Production I.G            NaN   
4               Production I.G            NaN   
...                        ...            ...   
18490                  Pierrot         Winter   
18491                      NaN            NaN   
18492  Tomoyasu Murata Company            NaN   
18493                      NaN            NaN   
18494                      NaN            NaN   

                                                    Tags  Rating  \
0                                    Promotional, Shorts    3.41   
1      Romance, Library, School Life, Based on a Ligh...    3.40   
2      Drama, Romance, Episodic, School Life, Based o...    3.69   
3      Drama, Mystery, Romance, School Club, School L...    3.74   
4      Shounen, Sports, Boxing, Hand to Hand Combat, ...    2.98   
...                                                  ...     ...   
18490  Action, Comedy, Sci Fi, Shounen, Aliens, Body ...    2.91   
18491                                     Dancing, Idols    3.46   
18492                      Abstract, No Dialogue, Shorts     NaN   
18493                      Shorts, Stop Motion Animation     NaN   
18494                   No Dialogue, School Life, Shorts    3.05   

       Release_year  End_year  \
0            2022.0       NaN   
1            2009.0       NaN   
2            2010.0       NaN   
3            2010.0       NaN   
4            1990.0       NaN   
...             ...       ...   
18490        2017.0       NaN   
18491        2019.0       NaN   
18492        2005.0       NaN   
18493        1998.0       NaN   
18494        2019.0       NaN   

                                             Description Content_Warning  \
0      'The anime depicts the "smart city" envisioned...             NaN   
1      'Konoha sometimes sees Touko on the street, re...             NaN   
2                   'No synopsis yet - check back soon!'             NaN   
3      "As first year student Konoha Inoue is about t...             NaN   
4      Leg

In [ ]:
df_anime.sort_values(by=['Related_Mange', 'Release_year']).reset_index()

index   Rank                                            Name  \
0      14008  14009  Amai Choubatsu - Watashi wa Kanshu Sen'you Pet   
1      11954  11955                                  "Eiyuu" Kaitai   
2      17940  17941                         Shounen Sunday CM: LOVe   
3       6392   6393           Daitai 3-pun de Wakaru: .hack History   
4       6399   6400                        .hack//SIGN - Intermezzo   
...      ...    ...                                             ...   
18490  18457  18458                                   Zhuangzi Shuo   
18491  18461  18462             Hua Jianghu Zhi Gui Yexing: Part II   
18492  18462  18463          Jingju Mao 4 Zhi Jiaota Shidi: Part II   
18493  18479  18480                                  Baise Shandian   
18494  18487  18488                                     Make My Day   

                                           Japanese_name   Type  Episodes  \
0       At This Rate, I'm Gonna Come... -The Warden's...  Web        13.0   
1                                                    NaN  OVA         1.0   
2                                                    NaN  Other       NaN   
3                     Learn in 3 Minutes!! .hack History  Web         1.0   
4                                                    NaN  OVA         1.0   
...                                                  ...    ...       ...   
18490                                                NaN  TV         70.0   
18491                                                NaN  Web         NaN   
18492                            Jing-Ju Cats 4: Part II  Web         NaN   
18493                                                NaN  Web         NaN   
18494                                                NaN  Web         NaN   

                 Studio Release_season  \
0             Magic Bus         Spring   
1                 Zexcs            NaN   
2                   NaN            NaN   
3                   NaN            NaN   
4             BEE TRAIN            NaN   
...                 ...            ...   
18490               NaN            NaN   
18491     Beijing Rocen            NaN   
18492               NaN            NaN   
18493  HuaMei Animation            NaN   
18494          Studio 5            NaN   

                                                    Tags  Rating  \
0      Josei, Prison, Short Episodes, Smut, Stockholm...    2.84   
1      Drama, Fantasy, Slice of Life, Person in a Str...    3.09   
2      Sports, Commercials, Promotional, Based on a M...     NaN   
3            Comedy, Chibi, Recap, Based on a Video Game    3.43   
4                  Dungeon, MMORPG, RPG, Virtual Reality    3.43   
...                                                  ...     ...   
18490      Chinese Animation, Historical, Short Episodes     NaN   
18491  Action, Sports, Cars, Chinese Animation, Racin...     NaN   
18492  Action, Adventure, Fantasy, Animal Protagonist...     NaN   
18493               Sports, Chinese Animation, Ping Pong     NaN   
18494                    Horror, Monsters, Original Work     NaN   

       Release_year  End_year  \
0            2018.0       NaN   
1            2016.0       NaN   
2               NaN       NaN   
3            2017.0       NaN   
4            2003.0       NaN   
...             ...       ...   
18490           NaN       NaN   
18491           NaN       NaN   
18492           NaN       NaN   
18493           NaN       NaN   
18494           NaN       NaN   

                                             Description  \
0      "You're dripping wet down here... Is the pat-d...   
1      Hakua and Kougyoku Tsuranari are retired heroe...   
2      A commercial for the Weekly Shounen Sunday man...   
3                   'No synopsis yet - check back soon!'   
4      'Bear and Mimiru are on the trail of a time-re...   
...                                                  ...   
18490                 No synopsis yet - check back soon!   
18491     The second part of H

In [ ]:
df_anime.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 18495 entries, 0 to 18494
Data columns (total 17 columns):
 #   Column           Non-Null Count  Dtype  
---  ------           --------------  -----  
 0   Rank             18495 non-null  int64  
 1   Name             18495 non-null  object 
 2   Japanese_name    7938 non-null   object 
 3   Type             18495 non-null  object 
 4   Episodes         9501 non-null   float64
 5   Studio           12018 non-null  object 
 6   Release_season   4116 non-null   object 
 7   Tags             18095 non-null  object 
 8   Rating           15364 non-null  float64
 9   Release_year     18112 non-null  float64
 10  End_year         2854 non-null   float64
 11  Description      18491 non-null  object 
 12  Content_Warning  1840 non-null   object 
 13  Related_Mange    7627 non-null   object 
 14  Related_anime    10063 non-null  object 
 15  Voice_actors     15309 non-null  object 
 16  staff            13005 non-null  object 
dtypes: float64(4

In [ ]:
# criando a função de processamento dos dados
def processamento_df(df_anime):

    # 3 primeiras linhas do df
    df_3_linhas = df_anime.head(3)

    # selecionando aleatoriamente 100 linhas do df
    df_menor = df_anime.sample(100)

    # removendo colunas com muitos dados nulos
    df_anime = df_anime.drop(columns='staff')

    # ordenando por conteúdo sensível
    df_anime = df_anime.sort_values("Content_Warning")

    # removendo dados duplicados
    df_anime = df_anime.drop_duplicates()

    # reset index
    df_anime = df_anime.reset_index()

    # lista de colunas
    colunas = list(df_anime.columns)

    # metricas: analise descritiva
    metricas = df_anime.describe(include='all')

    return df_3_linhas, df_menor, df_anime, colunas, metricas




In [ ]:
 # chamando a função processamento usando os dados do df
df_3_linhas, df_menor, df_teste, colunas, metricas = processamento_df(df_anime)

In [ ]:
# verificando se o retorno das 3 primeiras linhas está correto
df_3_linhas

Rank                                               Name  \
0     1  Demon Slayer: Kimetsu no Yaiba - Entertainment...   
1     2                     Fruits Basket the Final Season   
2     3                                    Mo Dao Zu Shi 3   

                    Japanese_name   Type  Episodes             Studio  \
0   Kimetsu no Yaiba: Yuukaku-hen  TV          NaN           ufotable   
1         Fruits Basket the Final  TV         13.0  TMS Entertainment   
2      The Founder of Diabolism 3  Web        12.0   B.C MAY PICTURES   

  Release_season                                               Tags  Rating  \
0          Fall   Action, Adventure, Fantasy, Shounen, Demons, H...    4.60   
1         Spring  Drama, Fantasy, Romance, Shoujo, Animal Transf...    4.60   
2            NaN  Fantasy, Ancient China, Chinese Animation, Cul...    4.58   

   Release_year  End_year                                        Description  \
0        2021.0       NaN  'Tanjiro and his friends accompany the Hashira...   
1        2021.0       NaN                  'The final arc of Fruits Basket.'   
2        2021.0       NaN               'The third season of Mo Dao Zu Shi.'   

                                     Content_Warning  \
0                                  Explicit Violence   
1  Emotional Abuse,, Mature Themes,, Physical Abu...   
2                                                NaN   

                                       Related_Mange  \
0                     Demon Slayer: Kimetsu no Yaiba   
1               Fruits Basket, Fruits Basket Another   
2  Grandmaster of Demonic Cultivation: Mo Dao Zu ...   

                                       Related_anime  \
0  Demon Slayer: Kimetsu no Yaiba, Demon Slayer: ...   
1  Fruits Basket 1st Season, Fruits Basket 2nd Se...   
2                   Mo Dao Zu Shi 2, Mo Dao Zu Shi Q   

                                        Voice_actors  \
0  Inosuke Hashibira : Yoshitsugu Matsuoka, Nezuk...   
1  Akito Sohma : Maaya Sakamoto, Kyo Sohma : Yuum...   
2  Lan Wangji, Wei Wuxian, Jiang Cheng, Jin Guang...   

                                               staff  
0  Koyoharu Gotouge : Original Creator, Haruo Sot...  
1  Natsuki Takaya : Original Creator, Yoshihide I...  
2  Mo Xiang Tong Xiu : Original Creator, Xiong Ke...

In [ ]:
# verificando se o retorno dos dados aleatorios das 100 linhas está correto
df_menor

Rank                                             Name  \
6299    6300                         Pokemon: Diamond & Pearl   
13531  13532           Atama wa Tsukaiyou. Card mo Tsukaiyou.   
7278    7279                              Asobotto Senki Goku   
17641  17642                                        Ten Count   
15934  15935                   ASTAR-reul Hyanghae Chaguchagu   
...      ...                                              ...   
11520  11521  Shiritsu Araiso Koutougakkou Seitokai Shikkoubu   
15765  15766                  Shenmi Shijie Lixian Ji Movie 4   
10742  10743   Mikasa Dai-senpai to Manabu: Anime "Azur Lane"   
16246  16247                      Kainar Asy-Tac Fronteer OVA   
924      925                                 Blast of Tempest   

                                       Japanese_name   Type  Episodes  \
6299                                             NaN  TV        191.0   
13531                                            NaN  Other       NaN   
7278                                  Monkey Typhoon  TV         52.0   
17641                                            NaN  Other       NaN   
15934           ASTAR Cup: The League of Super Kicks  TV         27.0   
...                                              ...    ...       ...   
11520                                            NaN  OVA         2.0   
15765                            Yugo & Lala Movie 4  Movie       NaN   
10742                                            NaN  TV Sp       NaN   
16246                                            NaN  OVA         4.0   
924     Zetsuen no Tempest: The Civilization Blaster  TV         24.0   

                         Studio Release_season  \
6299                        OLM          Fall    
13531                       NaN            NaN   
7278                 Studio Egg          Fall    
17641                       NaN            NaN   
15934                       NaN            NaN   
...                         ...            ...   
11520          Nippon Animation            NaN   
15765                       NaN            NaN   
10742  Bibury Animation Studios            NaN   
16246                       NaN            NaN   
924                       Bones          Fall    

                                                    Tags  Rating  \
6299   Adventure, Fantasy, Elemental Powers, Episodic...    3.43   
13531                   Comedy, Commercials, Promotional    2.92   
7278   Action, Adventure, Sci Fi, Shounen, Journey to...    3.38   
17641  BL, Romance, Yaoi, Adult Couples, LGBT Themes,...     NaN   
15934                   Sports, Korean Animation, Soccer     NaN   
...                                                  ...     ...   
11520                      School Life, Based on a Manga    3.13   
15765  Adventure, Fantasy, Chinese Animation, Family ...     NaN   
10742  Anthropomorphic, Educational, Military, Ocean,...    3.18   
16246  Action, Mecha, Sci Fi, Chinese Animation, CG A...     NaN   
924    Adventure, Drama, Mystery, Romance, Shounen, A...    3.99   

       Release_year  End_year  \
6299         2006.0    2010.0   
13531        2014.0       NaN   
7278         2002.0    2003.0   
17641           NaN       NaN   
15934        2014.0    2015.0   
...             ...       ...   
11520        2002.0       NaN   
15765        2018.0       NaN   
10742        2019.0       NaN   
16246        2015.0       NaN   
924          2012.0    2013.0   

                                             Description Content_Warning  \
6299   'In the Sinnou region, a young girl named Dawn...             NaN   
13531                 No synopsis yet - check back soon!             NaN   
7278   '1000 years in the future, both humans and Asu...             NaN   
17641  Corporate secretary Shirotani suffers from obs...             NaN   
15934                 No synopsis yet - check back soon!             NaN   
...                                                  ...             ...   
115

In [ ]:
# verificando se o retorno da lista de colunas
colunas

['index',
 'Rank',
 'Name',
 'Japanese_name',
 'Type',
 'Episodes',
 'Studio',
 'Release_season',
 'Tags',
 'Rating',
 'Release_year',
 'End_year',
 'Description',
 'Content_Warning',
 'Related_Mange',
 'Related_anime',
 'Voice_actors']

In [ ]:
# verificando se o retorno do tratamento dos dados está correto
df_teste

index   Rank                                               Name  \
0       2403   2404                                          Hell Girl   
1      13776  13777                         Ichi the Killer: Episode 0   
2        161    162                               Parasyte -the maxim-   
3      12559  12560                                   Nekojiru Gekijou   
4       1897   1898                           Revolutionary Girl Utena   
...      ...    ...                                                ...   
18490  18490  18491  Qin Shi Mingyue: Canghai Hengliu Xiaomeng Spec...   
18491  18491  18492                      Yi Tang Juchang: Sanguo Yanyi   
18492  18492  18493          Fenghuang Ji Xiang Yu Qingming Shanghe Tu   
18493  18493  18494                      Chengshi Jiyi Wo Men de Jieri   
18494  18494  18495  Heisei Inu Monogatari Bow: Genshi Inu Monogata...   

                              Japanese_name   Type  Episodes  \
0                             Jigoku Shoujo  TV         26.0   
1       Koroshiya-1 The Animation Episode 0  OVA         1.0   
2                Kiseijuu: Sei no Kakuritsu  TV         24.0   
3        Nekojiru Gekijou Jirujiru Original  TV         27.0   
4                      Shoujo Kakumei Utena  TV         39.0   
...                                     ...    ...       ...   
18490                                   NaN  Web         2.0   
18491                                   NaN  TV        108.0   
18492                                   NaN  TV         13.0   
18493                                   NaN  TV          NaN   
18494                                   NaN  Movie       NaN   

                             Studio Release_season  \
0                       Studio DEEN          Fall    
1                               AIC            NaN   
2                          MADHOUSE          Fall    
3                        Kent House         Summer   
4                         J.C.Staff         Spring   
...                             ...            ...   
18490  Sparkly Key Animation Studio            NaN   
18491                           NaN            NaN   
18492                           NaN            NaN   
18493                           NaN            NaN   
18494              Nippon Animation            NaN   

                                                    Tags  Rating  \
0      Horror, Deal with the Devil, Episodic, Melanch...    3.74   
1      Action, Drama, Fetish, Psychological, Psychopa...    2.88   
2      Action, Drama, Horror, Sci Fi, Seinen, Aliens,...    4.28   
3      Comedy, Animal Protagonists, Anthropomorphic, ...    3.04   
4      Drama, Fantasy, Boarding School, Conspiracy, C...    3.80   
...                                                  ...     ...   
18490  Action, Ancient China, Chinese Animation, Hist...     NaN   
18491                                  Chinese Animation     NaN   
18492  Chinese Animation, Family Friendly, Short Epis...     NaN   
18493  Chinese Animation, Family Friendly, Short Epis...     NaN   
18494                        Comedy, Slice of Life, Dogs     NaN   

       Release_year  End_year  \
0            2005.0    2006.0   
1            2002.0       NaN   
2            2014.0    2015.0   
3            1999.0       NaN   
4            1997.0       NaN   
...             ...       ...   
18490        2020.0       NaN   
18491        2010.0       NaN   
18492        2020.0       NaN   
18493        2020.0       NaN   
18494        1994.0       NaN   

                                             Description  \
0      'Has someone done something to hurt you or the...   
1      Hajime Shiroishi is a disturbed, troubled boy....   
2      "In secret, alien parasites drift downwards to...   
3      Cats can be very selfish and inconsiderate bei...   
4      "When Utena Tenjou was very little her parents...   
...                                                  ...   
18490  Special episodes of Qin Shi Mingyue: Canghai H...   
18491 

In [ ]:
# verificando se o retorno das metricas está correto
metricas.T


count unique  \
index            18495.0    NaN   
Rank             18495.0    NaN   
Name               18495  18443   
Japanese_name       7938   7902   
Type               18495      8   
Episodes          9501.0    NaN   
Studio             12018    745   
Release_season      4116      4   
Tags               18095  11799   
Rating           15364.0    NaN   
Release_year     18112.0    NaN   
End_year          2854.0    NaN   
Description        18491  10012   
Content_Warning     1840    208   
Related_Mange       7627   4151   
Related_anime      10063   9451   
Voice_actors       15309  13615   

                                                               top  freq  \
index                                                          NaN   NaN   
Rank                                                           NaN   NaN   
Name                                             [email protected]     4   
Japanese_name                                                  The    14   
Type                                                         TV     5446   
Episodes                                                       NaN   NaN   
Studio                                              Toei Animation   737   
Release_season                                              Spring  1291   
Tags                                                      Vocaloid   395   
Rating                                                         NaN   NaN   
Release_year                                                   NaN   NaN   
End_year                                                       NaN   NaN   
Description                     No synopsis yet - check back soon!  4579   
Content_Warning                                           Violence   529   
Related_Mange    Crayon Shin-chan, Shin Crayon Shin-chan, Crayo...    33   
Related_anime                                            One Piece    35   
Voice_actors                               Taku Furukawa\nDirector    42   

                        mean         std     min     25%     50%      75%  \
index                 9247.0  5339.19095     0.0  4623.5  9247.0  13870.5   
Rank                  9248.0  5339.19095     1.0  4624.5  9248.0  13871.5   
Name                     NaN         NaN     NaN     NaN     NaN      NaN   
Japanese_name            NaN         NaN     NaN     NaN     NaN      NaN   
Type                     NaN         NaN     NaN     NaN     NaN      NaN   
Episodes            20.92085   37.990858     1.0     2.0    12.0     26.0   
Studio                   NaN         NaN     NaN     NaN     NaN      NaN   
Release_season           NaN         NaN     NaN     NaN     NaN      NaN   
Tags                     NaN         NaN     NaN     NaN     NaN      NaN   
Rating              3.355133    0.400624    0.96    3.13    3.36     3.59   
Release_year     2006.520318   15.189537  1907.0  2001.0  2012.0   2017.0   
End_year         2004.256132   13.257484  1962.0  1996.0  2007.0   2015.0   
Description              NaN         NaN     NaN     NaN     NaN      NaN   
Content_Warning          NaN         NaN     NaN     NaN     NaN      NaN   
Related_Mange            NaN         NaN     NaN     NaN     NaN      NaN   
Related_anime            NaN         NaN     NaN     NaN     NaN      NaN   
Voice_actors             NaN         NaN     NaN     NaN     NaN      NaN   

                     max  
index            18494.0  
Rank             18495.0  
Name                 NaN  
Japanese_name        NaN  
Type                 NaN  
Episodes           800.0  
Studio               NaN  
Release_season       NaN  
Tags                 NaN  
Rating               4.6  
Release_year      2023.0  
End_year          2022.0  
Description          NaN  
Content_Warning      NaN  
Related_Mange        NaN  
Related_anime        NaN  
Voice_actors         NaN

In [ ]:
# salvando o df para um arquivo csv
df_teste.to_csv("df_anime_processado.csv")

# Insights:

- A avaliação ficou com a nota máxima de 4.6 e mínima de 0.96 com desvião padrão de 0.40062;
- O principal anime com conteúdo relacionado é One Piece;
- O alerta mais frequente de conteúdo sensível é sobre violência;
- O ano de lançamento mais antigo é 1907;
- o ano de encerramento mais antigo é 1962.





# Bônus:

* Traduzir colunas do dataset Anime do inglês para o português.





In [ ]:
dict_tradutor = {
' Rank'  : ' Posição',
'Name' : 'Nome',
'Japanese_name' : 'Nome em japonês',
'Type' : 'Tipo',
'Episodes' : 'Episódios',
'Studio' : 'Estudio',
'Realese_season' : 'Temporada',
'Tags' : 'Etiquetas',
'Rating' : 'Avaliação',
'Release_year' : 'Ano de lançamento',
'End_year' : 'Ano de encerramento',
'Description' : 'Descrição',
'Content_warning' : 'Aviso de conteúdo sensível',
'Related_mange' : 'Manga relacionado (Erro de tipografia)',
'Related_anime' : 'Anime relacionado',
'Voice_actors' : 'Dubladores',
'Staff' : 'Equipe de produção'
}


colunas_portugues = []

for chave, valor in dict_tradutor.items():
    colunas_portugues.append(valor)

colunas_portugues

[' Posição',
 'Nome',
 'Nome em japonês',
 'Tipo',
 'Episódios',
 'Estudio',
 'Temporada',
 'Etiquetas',
 'Avaliação',
 'Ano de lançamento',
 'Ano de encerramento',
 'Descrição',
 'Aviso de conteúdo sensível',
 'Manga relacionado (Erro de tipografia)',
 'Anime relacionado',
 'Dubladores',
 'Equipe de produção']

In [ ]:
dict_tradutor.items()

dict_items([(' Rank', ' Posição'), ('Name', 'Nome'), ('Japanese_name', 'Nome em japonês'), ('Type', 'Tipo'), ('Episodes', 'Episódios'), ('Studio', 'Estudio'), ('Realese_season', 'Temporada'), ('Tags', 'Etiquetas'), ('Rating', 'Avaliação'), ('Release_year', 'Ano de lançamento'), ('End_year', 'Ano de encerramento'), ('Description', 'Descrição'), ('Content_warning', 'Aviso de conteúdo sensível'), ('Related_mange', 'Manga relacionado (Erro de tipografia)'), ('Related_anime', 'Anime relacionado'), ('Voice_actors', 'Dubladores'), ('Staff', 'Equipe de produção')])

In [ ]:
#Trabalho realizado em dupla